contracts/IERC721M.sol

```diff
@@ -25,22 +25,22 @@ interface IERC721M {
     error WithdrawFailed();
 
     struct MintStageInfo {
-        uint256 price;
+        uint80 price;
         uint32 walletLimit; // 0 for unlimited
         bytes32 merkleRoot; // 0x0 for no presale enforced
-        uint256 maxStageSupply; // 0 for unlimited
-        uint256 startTimeUnixSeconds;
-        uint256 endTimeUnixSeconds;
+        uint24 maxStageSupply; // 0 for unlimited
+        uint64 startTimeUnixSeconds;
+        uint64 endTimeUnixSeconds;
```

Comment:
looking good, I think the writer is from the owner, for double checking the boundary of these fields